peachjam/models/judgment.py

```diff
@@ -51,10 +51,10 @@ def __str__(self):
 
 class Judgment(CoreDocument):
     court = models.ForeignKey(Court, on_delete=models.PROTECT, null=True)
-    judges = models.ManyToManyField(Judge, blank=True)
-    headnote_holding = models.TextField(blank=True)
-    additional_citations = models.TextField(blank=True)
-    flynote = models.TextField(blank=True)
+    judges = models.ManyToManyField(Judge, null=True, blank=True)
```

Comment:
I don't think `null` has any effect on ManyToManyField. blank=true should be sufficient.